unittest/map.cpp

```diff
@@ -70,6 +78,7 @@ void ARegionList::MakeRegions(int level, int xSize, int ySize)
 				
 				Add(reg);
 				arr->SetRegion(x, y, reg);
+				Adot();
```

Comment:
Just to make it easy to see how many regions got created since there are only 4 and it made testing the cout capturing have some data.